movo_common/movo_third_party/moveit_python/scripts/dump_planning_scene.py

```diff
@@ -45,7 +45,7 @@
                          PlanningSceneComponents.ROBOT_STATE_ATTACHED_OBJECTS
         scene = service(req)
     except rospy.ServiceException as e:
-        print("Failed to get planning scene: %s" % e)
+        print(("Failed to get planning scene: %s" % e))
```

Comment:
Lots of extra bracket in total

Reply:
This is an artifact of 2to3, because the argument of the print function are found to be either a tuple or a formatted string. Given that there are currently 49 files, and these print statements would all need to be reverted manually, I am considering allowing the extra brackets because the code runs with them in.

Reply:
its not good syntax. you can just fix manually. can also just use smart regex to do it quick